Stratis.Bitcoin/Features/BlockStore/BlockStoreLoopStep.cs

```diff
@@ -33,6 +54,12 @@ internal void SetNextStep(BlockStoreLoopStep step)
         }
     }
 
+    /// <summary> 
+    /// Base class for each BlockStoreLoopStep
+    /// <para>
+    /// Each step must implement ExecuteAsync passing in the next chained block to process
```

Comment:
This sounds like trivial interpretation of the abstract keyword on the ExecuteAsync method. What ExecuteAsync does should be described in its comment and the fact that it must be implemented is given by using abstract keyword.